.circleci/config.yml

```diff
@@ -175,6 +184,35 @@ jobs:
           token_name: JIRA_CIRCLE_TOKEN
           job_type: build
 
+  pact_check_and_publish:
+    parameters:
+      consumer_tags:
+        type: string
+    environment:
+      PACTBROKER_HOST: "pact-broker-prod.apps.live-1.cloud-platform.service.justice.gov.uk"
+    executor: builder
+    steps:
+      - checkout
+      - restore_cache:
+          keys:
+            - gradle-{{ checksum "build.gradle" }}
+            - gradle-
+      - run:
+          command: |
+            PACT_PROVIDER_VERSION="$CIRCLE_SHA1" \
+              PACT_PROVIDER_TAG="$CIRCLE_BRANCH" \
```

Comment:
Can we/should we use APP_VERSION for this so it matches the docker image tag?

Reply:
You mean for the PACT_PROVIDER_TAG ? I am not sure. As I understand it, this allows us to see what is happening on the branch as it is committed. 
Or are you talking about the PACT_PROVIDER_VERSION ?

Reply:
Updated